src/rust/plugin-registry/src/server/s3_url.rs

```diff
@@ -1,13 +1,11 @@
 use grapl_config::env_helpers::ENV_ENDPOINT;
 
-/// Some discussion about this function here:
-/// https://grapl-internal.slack.com/archives/C02J5JYS92S/p1648667074625629
 pub fn get_s3_url(bucket: &str, key: &str) -> String {
     let local_endpoint = std::env::var(ENV_ENDPOINT).ok();
     // If the above is specified, we're running locally against Localhost
     // Otherwise, it's against prod S3; so we can use virtual-hosted-style URLs
```

Comment:
This comment looks like it needs updating now.

Reply:
Updated. Unfortunately, since I can't access the slack message, the comment I've added may be incomplete but 🤷 .
Let me know if there's something you think I should add.

Reply:
I was mainly referring to the fact that it's not an S3 virtual-host-style URL anymore (and not even a URL, or even a URI, I think :shrug:)

If it's all about how `go-getter` handles things, then a mention of that and any relevant links would be useful.

Without that kind of background information, I could _easily_ see someone in the future "helpfully" refactoring this into something that doesn't work anymore.

Reply:
Updated docs again. This should be better